dashboard/app/models/unit_group.rb

```diff
@@ -49,6 +49,7 @@ def skip_name_format_validation
     is_stable
     visible
     pilot_experiment
+    unit_group_announcements
```

Comment:
I appreciate the enthusiastic adoption of the new "unit group" name, but I think this should just be `announcements`. It is already a property on the `UnitGroup` model, it should be clear that it is a unit group announcement and not a script announcement.

Reply:
Ya I would prefer this be announcements too. I went with `unit_group_announcements` because the one on script is `script_announcements` and the consistency seemed worth it. Another option is to fix script to be `announcements`. Thoughts? @davidsbailey

Reply:
I see, makes sense! I like changing both to announcements.